tool/check-shell-scripts

```diff
@@ -0,0 +1,73 @@
+#!/bin/sh
+# tool/check-shell-scripts -- check shell scripts in the MPS
+#
+# Copyright (c) 2023 Ravenbrook Limited. See end of file for license.
+#
+# This script finds shell scripts in the MPS tree and runs shellcheck
+# on them to check for issues.
+#
+# It can be invoked from the command line of from Continuous
+# Integration scripts.
```

Comment:
How? Any parameters? What to type to get usage?

Reply:
What assumptions are made about where the script is invoked from (i.e. location within a directory) in order for the script to perform it's function correctly

Reply:
> m: How? Any parameters? What to type to get usage?

Fix: Clarified usage in bd0c035c5.  Decided not to add "--help" or similar for this simple script.

Reply:
> m: What assumptions are made about where the script is invoked from (i.e. location within a directory) in order for the script to perform its function correctly

Fix: Clarified where the script should be run in bd0c035c5.